meta-spec-0001.md

```diff
@@ -34,9 +34,9 @@ They do, however,
 {{< gallery >}}
 
 {{< person
-      name="?"
-      github_handle=""
-      avatar=""
+      name="Matthias Bussonnier"
+      github_handle="carreau"
+      avatar="https://avatars.githubusercontent.com/u/335567?s=200"
       projects="ipython"
```

Comment:
We are bootstrapping by choosing people from projects.  But you should add any other relevant projects that you are working on separated by a comma.